pest-ion/tests/cli.rs

```diff
@@ -29,14 +71,19 @@ enum FileMode {
             }
         }
     "#
-)]
-fn run_it(
-    #[case] pest_src: &str,
-    #[case] ion_text: &str,
+).into())]
+#[case::partiql(partiql_test_case())]
+fn run_it<S: AsRef<str>>(
```

Comment:
Is this `S: AsRef<str>` doing anything? I see `as_ref()` is called on `pest_text` below, but that's already a string.

Reply:
Yes, in some cases we throw `&'static str` in the case where we load PartiQL's grammar, we pass in a `String`, so this just normalizes the test case input.